libs/designsystem/src/lib/components/data-table/table-row/table-row.component.ts

```diff
@@ -0,0 +1,16 @@
+import { ChangeDetectorRef, Component, HostBinding, Input, OnInit } from '@angular/core';
+
+@Component({
+  // eslint-disable-next-line @angular-eslint/component-selector
+  selector: 'tr[kirby-tr]',
+  template: ` <ng-content></ng-content> `,
+})
+export class TableRowComponent implements OnInit {
+  @HostBinding('class.kirby-selectable-row') @Input() selectable: boolean = false;
+
+  constructor(private readonly cdRef: ChangeDetectorRef) {}
+
+  ngOnInit(): void {
+    this.cdRef.detach();
```

Comment:
I'm fairly certain that change detection is run automatically when an Input() value is changed - and this would cover that situation? The conclusion of this article explains, but I may be wrong? https://blog.angular-university.io/onpush-change-detection-how-it-works/

Reg. encapsulation: From angular docs: "If the policy is ViewEncapsulation.Emulated and the component has no styles nor styleUrls, the policy is automatically switched to ViewEncapsulation.None." - But I changed the encapsulation for clarity :).